cdm_reader_mapper/core/databundle.py

```diff
@@ -341,19 +343,26 @@ def copy(self):
             setattr(db, key, value)
         return db
 
-    def select_true(self, mask=False, inplace=False, **kwargs):
+    def select_true(self, mask=False, return_invalid=False, inplace=False, **kwargs):
```

Comment:
@jtsiddons: I added a new parameter to the select_* functions: ``return_invalid``

If this is True, the functions return an extra pd.DataFrame containing the unselected entries in addition to the DataBundle, which is always returned.

I am not sure if I am really happy with this "solution".

Reply:
@jtsiddons: I made some improvements:

- if ``inplace`` is ``True`` select functions return ``None`` to make it consistent with pandas
- new function ``select_false`` which selects **in**valid values from data via mask 
- parameter ``mask`` is  no longer available; instead selection method is applied to mask as to data
- parameter ``return_invalid`` is no longer available; select functions return DataBundle containing selected data
- new parameter ``inverse``: select function return DataBundle containing **un**selected data (is inverse good wording?). E.g. ``select_true(inverse=True)`` returns the same result as ``select_false()``

Reply:
- select function do NOT reset indexes by default. ONe can opyitonally set this with parameter ``reset_index``.